docs/aks/onboarding-form.md

```diff
@@ -0,0 +1,45 @@
+# Teacher services infra - Service onboarding form
+
+Note - this is a migration of two existing services. They share a single codebase, are served to the public from
+two different domains, and are currently deployed to (non-AKS) Azure.
+
+- Service name? Access your teaching qualifications / Check a teacher's record (two services, single codebase)
+- Domains ?
+  - access-your-teaching-qualifications.education.gov.uk (already exists)
+  - check-a-teachers-record.education.gov.uk (already exists)
+  - access-your-teaching-qualifications.teacherservices.cloud
+  - check-a-teachers-record.teacherservices.cloud
+  - test._ and preprod._ variants of the above.
+- Service short name? aytqctr
+- Repository? https://github.com/DFE-Digital/access-your-teaching-qualifications/
+- Environments? e.g. develoment, staging, production
+  - test
+  - preprod
+  - production
+- Review apps? Yes or No
+  - Yes
+- Cost centre/Activity code ? e.g. 10167/101304
+  - Unknown
+- List of developers who will need access to Azure? Do they use a DfE laptop or BYOD? e.g
+  - Felix Clack - BYOD
+  - James Gunn - BYOD
+  - Richard Pattinson - DfE laptop
```

Comment:
@malcolmbaig you are owner of the `s189 TRA delivery team` group. You can add Richard as member